contributing/README.md

```diff
@@ -104,20 +104,114 @@ Signed-off-by: Random J Developer <random@developer.example.org>
 Git even has a -s command line option to append this automatically to your
 commit message:
 
-```
-$ git commit -s -m 'This is my commit message'
+```bash
+git commit -s -m 'This is my commit message'
 ```
 
 If you have already made a commit and forgot to include the sign-off, you can
 amend your last commit to add the sign-off with the following command, which can
 then be force pushed.
 
-```
+```bash
 git commit --amend -s
 ```
 
 We use a [DCO bot] to enforce the DCO on all commits in every pull request.
 
+## Code Review Process
+
+All Pull Requests (PR), whether written by a Crossplane maintainer or a
+community member, must go through code review.
+
+Not only do code reviews ensure that the code is correct, maintainable, and
+secure, but more importantly it allows us to use code reviews as an educational
+tool.
+
+With this in mind, all efforts around code reviews should be seen through the
+lens of educating the reviewee and be accompanied by kind, detailed feedback
+that will help reviewees understand the context the reviewer is coming from.
+
+We encourage anyone in the community to conduct a code review on a PR.
+In most situations we prefer to have the following approvals before
+merging a PR:
+
+* At least one approval from [Reviewers]
+* At least one approval from [Maintainers]
+
+When opening a PR, GitHub will assign reviewers based on the project's
+code review settings and who gets assigned depends on what code the contributor
+changed, per [CODEOWNERS]. In most cases we expect that someone from
+[crossplane-reviewers] and a subject matter expert from
+[crossplane-maintainers] will be assigned.
+
+We encourage reviews from the community and [Reviewers] to take place before
+someone from the [Maintainers] group reviews a PR. This helps reduce the load
+on the project maintainers and ensures they can be more efficient in their
+reviews.
+
+### Expectation of PR Authors
+
+When your PR is ready for review please notify the assigned reviewers by
+by mentioning them in a comment.
+
+After implementing review feedback, the PR author should notify the reviewer
+by mentioning them in a comment when the PR is ready for another
+review.
+
+If you are not getting a response within a reasonable timeframe, remembering
+that reviewers and maintainers offer their time free of charge and have other
+obligations, you can reach out to the `#crossplane-owners` channel in the
+Crossplane community [Slack] workspace.
+
+### Expectation of Reviewers
+
+If you are assigned as a reviewer on a PR and you are unable to commit to
+reviewing the PR within a reasonable timeframe, you are encouraged to
+communicate this and manage the PR author's expectation.
+
+All reviewers are encouraged to consider the following aspects and to provide
+guidance to the PR author before giving their approval:
+
+* Is the code functionally correct?
```

Comment:
possibly in this list we can reinforce "Are the changes well documented and their intent explained sufficiently for current and future readers of your code to easily understand it?", or something to that effect :)

Reply:
Adding a slightly more brief version: "Are the changes well documented, and their intent explained sufficiently for current and future readers?"